### PR TITLE
[Calling] Fix Settings for Audio Only

### PR DESCRIPTION
**Pull request by ahammer**
## Purpose

* Fix settings menu
* auto-token w/azure cli (time saving) (disabled)

Auto Token will fetch and install a token as if it were in your local.properties if
1) You have `az` azure cli installed
2) You have run `az login`
3) You have set `AZURE_COMMUNICATION_CONNECTION_STRING` environment variable to a valid Azure Communication Resource (Available in `keys` via the azure.portal.com).
4) You have omitted your ACS_TOKEN from your local.properties

If all the above is true, Tokens will be fetched and installed for you upon installation of the demo.

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull Request Checklist

- [ ] Public API changes
- [ ] Verified for cross-platform
- [ ] Synced with iOS, Web team
- [ ] Internal review completed
- [ ] UI Changes
  - [ ] Screen captures included
  - [ ] Tested for Light/Dark mode
  - [ ] Tested for screen rotation
  - [ ] Tested on Tablet and small screen device (5")
  - [ ] Include localization changes
- [ ] Tests
  - [ ] Memory leak analysis performed
  - [ ] Tested on API 21, 26 and latest 
  - [ ] Tested for background mode
  - [ ] Unit Tests Included
  - [ ] UI Automated Tests Included

## How to Test

* Open...
* Click on...

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by pavelprystinka — DISMISSED** [review]

**Review by pavelprystinka — APPROVED** [review]

File: azure-communication-ui/auto_token.gradle [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Install AZ CLI and ensure the command below can run.(Setup the Environment)
+import groovy.json.JsonSlurper
```
Comment (iaulakh) [inline azure-communication-ui/auto_token.gradle:2]: this gradle can be moved inside demo app? as only demo app requires?

**Review by pavelprystinka — APPROVED** [review]

**Review by iaulakh — APPROVED** [review]

**Comment by iaulakh** [comment]
@ahammer ,
Re: I think it will be good idea to have separate repo for token help? perhaps that web and iOS can also leverage. May be having a discussion with PM to have repo and CLI instructions can help. 

You have az azure cli installed
You have run az login
You have set AZURE_COMMUNICATION_CONNECTION_STRING environment variable to a valid Azure Communication Resource (Available in keys via the azure.portal.com).

**Comment by ahammer** [comment]
> @ahammer , Re: I think it will be good idea to have separate repo for token help? perhaps that web and iOS can also leverage. May be having a discussion with PM to have repo and CLI instructions can help.
> 
> You have az azure cli installed You have run az login You have set AZURE_COMMUNICATION_CONNECTION_STRING 

https://learn.microsoft.com/en-us/cli/azure/communication/identity/token?view=azure-cli-latest
The instructions for setting up and utilizing Azure CLI for communication services token's is on that page.

The usage instructions for the Android Readme will be done after, when merging back to Develop I can add to the Readme.md under the "getting started" for android and reference/link that it uses that method under the hood to work.

This gradle function doesn't apply to iOS or Web directly, but I'd encourage the use of Azure CLI when it logically makes sense, as in this case.

**Review by iaulakh — APPROVED** [review]

File: azure-communication-ui/build.gradle [hunk 2]
```diff
@@ -111,6 +111,7 @@ plugins {
     id "org.jlleitschuh.gradle.ktlint" version "10.2.1" apply(true)
 }
 
+
```
Comment (jamchengms) [inline azure-communication-ui/build.gradle:114]: revert

File: azure-communication-ui/demo-app/build.gradle [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import groovy.json.JsonSlurper
```
Comment (jamchengms) [inline azure-communication-ui/demo-app/build.gradle:1]: revert

**Review by jamchengms — CHANGES_REQUESTED** [review]

**Review by jamchengms — APPROVED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Pull request merged**